modules/gdscript/language_server/gdscript_workspace.h

```diff
@@ -75,9 +75,12 @@ class GDScriptWorkspace : public Reference {
 
 	String get_file_path(const String &p_uri) const;
 	String get_file_uri(const String &p_path) const;
+	String simplify_file_uri(const String &p_path) const;
 
 	void publish_diagnostics(const String &p_path);
 	void completion(const lsp::CompletionParams &p_params, List<ScriptCodeCompletionOption> *r_options);
+	Error get_signature_help(const lsp::TextDocumentPositionParams &p_params, String *r_signature_name, String *r_signature_doc, List<String> *r_signature_parameter, int *r_cur_active_parameter);
+	Error get_document_link(const lsp::DocumentLinkParams &p_params, List<lsp::DocumentLink> *r_links_parameter);
```

Comment:
I don't think a path can link to more than one file so why need this function returns multi-values?

Reply:
Oh I'm wrong.
The LSP require us to return a `DocumentLink[]` so we have to search all the strings in scripts.